Mergin/validation.py

```diff
@@ -253,9 +253,10 @@ def check_value_relation(self):
                 ws = layer.editorWidgetSetup(i)
                 if ws and ws.type() == "ValueRelation":
                     cfg = ws.config()
-                    child_layer = next((l for l in self.layers.values() if l.source() == cfg["LayerSource"]), None)
+                    child_layer = next((v for k, v in self.layers.items() if k == cfg["Layer"]), None)
```

Comment:
are we sure cfg always has this key? or for safety add try..catch keyerror?

Reply:
Yes, these keys should be there. Otherwise value relation will be not configured and will not work. Maybe it is good idea to check this too, though I'm not sure user will leave it in a such broken state.